src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/util/CustomTimestampDeserialiser.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.util;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.text.ParseException;
+
+public class CustomTimestampDeserialiser extends StdDeserializer<Timestamp> {
```

Comment:
Could you please add a java doc comment on this class with an example of date format ?